VOTING_MEMBERS.md

```diff
@@ -17,38 +21,86 @@ The Voting Members, in alphabetic order by their GitHub handles, are:
 
 [@bdon](https://github.com/bdon)
 
+[@Binabh](https://github.com/Binabh) (Tribhuwan University)
+
 [@birkskyum](https://github.com/birkskyum)
 
+[@boldtrn](https://github.com/boldtrn) (Kurviger)
+
 [@brendan-ward](https://github.com/brendan-ward) (self-employed, some time covered by client contracts)
 
 [@chippieTV](https://github.com/chippieTV)
 
+[@ChrisLoer](https://github.com/ChrisLoer) (Felt)
+
+[@danielgard](https://github.com/danielgard) (komoot)
+
 [@dayjournal](https://github.com/dayjournal) (Mierune)
 
 [@drwestco](https://github.com/drwestco) (Microsoft)
 
+[@duje](https://github.com/duje)
+
+[@Fabi755](https://github.com/Fabi755)
+
 [@fredj](https://github.com/fredj) (Camptocamp)
 
+[@fynngodau](https://github.com/fynngodau)
+
 [@HarelM](https://github.com/harelm)
 
+[@HandyMenny](https://github.com/HandyMenny)
+
 [@hoeflehner](https://github.com/hoeflehner) (Toursprung)
 
+[@hy9be](https://github.com/hy9be) (AWS)
+
+[@ianthetechie](https://github.com/ianthetechie) (Stadia Maps)
+
+[@ibesora](https://github.com/ibesora) (Felt)
+
+[@IvanSanchez](https://github.com/IvanSanchez)
+
 [@JannikGM](https://github.com/JannikGM) (Graphmasters GmbH)
 
+[@jleedev](https://github.com/jleedev)
+
+[@jonahadkins](https://github.com/jonahadkins) (Meta)
+
+[@JonasVautherin](https://github.com/JonasVautherin) (Auterion)
+
 [@Joxit](https://github.com/Joxit) (Jawg Maps)
 
+[@JulianBissekkou](https://github.com/JulianBissekkou) (Tapped.dev)
+
 [@jutaz](https://github.com/jutaz)
 
+[@kajkal](https://github.com/kajkal)
+
 [@Kanahiro](https://github.com/Kanahiro) (Mierune)
 
+[@kevinschaul](https://github.com/kevinschaul)
+
 [@klokan](https://github.com/klokan) (MapTiler)
 
+[@louwers](https://github.com/louwers) (MapLibre)
```

Comment:
```suggestion
[@louwers](https://github.com/louwers)
```

Since I am not employed by MapLibre (and it is not a legal entity), I suggest we leave this out.